### PR TITLE
[Review][Teaser] Promote Teaser component from sandbox #190

### LINKED ISSUE #190: [Review][Teaser] Promote Teaser component from sandbox
With https://github.com/Adobe-Marketing-Cloud/aem-core-wcm-components/issues/155, the teaser component was updated to support:
- multiple Call-to-Action(s)
- image
- text
- rich-text description
- configuration based on content policies

I would like to call out a review of the component before we move it from sandbox to the main package.

### PR DESCRIPTION
**Pull request by vladbailescu**
* Add Teaser to master branch

Fixes #190

### CODE REVIEW

File: content/src/content/META-INF/vault/filter.xml [hunk 1]
```diff
@@ -16,4 +16,5 @@
   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~-->
 <workspaceFilter version="1.0">
     <filter root="/apps/core/wcm/components"/>
+    <filter root="/apps/core/wcm/sandbox/components"/>
```
Comment (jckautzmann) [inline content/src/content/META-INF/vault/filter.xml]: Is this needed on `master`?

File: bundles/core/src/main/java/com/adobe/cq/wcm/core/components/sandbox/models/package-info.java [hunk 2]
```diff
@@ -0,0 +1,40 @@
+/*~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```
Comment (jckautzmann) [inline bundles/core/src/main/java/com/adobe/cq/wcm/core/components/sandbox/models/package-info.java]: Isn't the `sandbox` folder only used on the `development` branch?

File: bundles/core/src/main/java/com/adobe/cq/wcm/core/components/sandbox/models/package-info.java [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+ *      {@link org.osgi.annotation.versioning.ConsumerType}s. An {@link java.lang.UnsupportedOperationException} could be thrown when a
+ *      component script would start using the newer API, without the actual implementation to support it. This can happen when an
+ *      implementor migrates a
+ *      <a href="https://helpx.adobe.com/experience-manager/core-components-v1/using/guidelines.html#ProxyComponentPattern">proxy component
+ *      </a> to a newer version of the core component it proxies and a custom Sling Model implementation, supporting an older API
+ *      version, is bound to this proxy component resource type.
+ * </p>
+ */
+@Version("1.0.0")
+package com.adobe.cq.wcm.core.components.sandbox.models;
```
Comment (jckautzmann) [inline bundles/core/src/main/java/com/adobe/cq/wcm/core/components/sandbox/models/package-info.java]: Isn't `sandbox` only used on the `development` branch?

**Comment by vladbailescu** [comment]
@jckautzmann Removed `sandbox` references, they were leftovers from the merge that I did not notice. Thanks!

**Pull request merged**